themes/godotengine/pages/fbx-import.htm

```diff
@@ -0,0 +1,106 @@
+  <a href="https://en.wikipedia.org/wiki/GlTF">glTF</a> is an open 3D
+  interchange format. Unlike FBX, it has many open source implementations
+  available and is supported out of the box in Godot.
+</p>
+<p>
+  We recommend using glTF over FBX for several reasons:
+  <ul>
+    <li>glTF is better supported in open source 3D software such as Blender.</li>
+    <li>
+      glTF import is integrated in the Godot editor. This way, team members
+      working on your project don't need to install external tools before they
+      can start working on the project.
+    </li>
+    <li>
+      If you use headless exporting (continuous integration) for your project,
+      you don't need to install external tools before you can export the
+      project.
+    </li>
+    <li>
+      If you use the Android or HTML5 editor, you can import glTF scenes, but
+      not FBX scenes as these platforms don't allow running arbitrary external
+      executables.
+    </li>
+    <li>
+      If you use a device on a non-x86 architecture, you can import glTF scenes,
+      but not FBX scenes as the Autodesk FBX SDK is only available for x86
+      architectures.
+    </li>
+    <li>
+      glTF is an open standard. Unlike FBX, it doesn't force other software to
+      link against a proprietary SDK to reliably import 3D scenes.
+    </li>
+  </ul>
+<p>
+  See <a href="https://godotengine.org/article/we-should-all-use-gltf-20-export-3d-assets-game-engines">this article</a> for more information.
+</p>
+
+<h3 class="title"">Download FBX2glTF</h3>
+<p><strong>
+  By downloading FBX2glTF, you accept the
+  <a href="https://nightly.link/godotengine/FBX2glTF/workflows/build.yaml/master/thirdparty-license.rtf.zip">Autodesk LICENSE AND SERVICES AGREEMENT for Autodesk FBX SDK 2020</a>
+  as FBX2glTF includes this proprietary FBX SDK.
+</strong></p>
+
+<p>
+  <a href="https://github.com/facebookincubator/FBX2glTF">Upstream FBX2glTF</a> is not maintained anymore, so the <a href="https://github.com/godotengine/fbx2gltf">Godot Engine fork of FBX2glTF</a>
+  must be used instead.
+</p>
+<p>
+  Here are download links for the latest version of Godot Engine's FBX2glTF fork:
+</p>
+
+  <div style="margin: 2rem 0">
+    <a href="https://nightly.link/godotengine/FBX2glTF/workflows/build.yaml/master/FBX2glTF.exe.zip" class="btn btn-default" style="margin: 0 0.5rem; white-space: nowrap">
+      Windows (x86_64)
+    </a>
+    <a href="https://nightly.link/godotengine/FBX2glTF/workflows/build.yaml/master/macos-packages.zip" class="btn btn-default" style="margin: 0 0.5rem; white-space: nowrap">
+      macOS (x86_64)
+    </a>
+    <a href="https://nightly.link/godotengine/FBX2glTF/workflows/build.yaml/master/FBX2glTF.zip" class="btn btn-default" style="margin: 0 0.5rem; white-space: nowrap">
+      Linux (x86_64)
+    </a>
+  </div>
+
+<p>
+  Support for more architectures such as ARM is planned in the future, but it
+  depends on Autodesk adding support for ARM in the FBX SDK first.
+</p>
```

Comment:
The FBX SDK already seems to provide universal binaries for macOS, so we could possibly do the same if FBX2glTF supports ARM64 properly.

For Windows/Linux, we could add x86_32 binaries, but indeed for ARM64 I don't see it happening.